e2e/explorer/api/src/app/etherscan/account.ts

```diff
@@ -91,3 +112,30 @@ export function balanceHistory(params: GetParams<BalanceHistory>) {
   if (!balance) return '0';
   return balance.toString();
 }
+
+export function tokenTx(params: GetParams<TokenTx>) {
+  const {address, contractAddress, startblock = 0, endblock, page, sort='asc', offset} = params;
+  if (!address || !contractAddress) throw new Error('Error! Missing address or contract address');
+  const transferID = id('Transfer(address,address,uint256)');
+
+  const sorting = {
+    asc: (a: TransactionResponse, b: TransactionResponse) => a.blockNumber - b.blockNumber,
+    desc: (a: TransactionResponse, b: TransactionResponse) => b.blockNumber - a.blockNumber
+  };  
+
+  // get the address transactions logs
+  const logs = store.logs[address]
+    .filter(log => {
+      if (log.address !== address) return false;
```

Comment:
As I mentioned in my previous comment, you don't need to filter the address anymore because you already get the logs from `store.logs[address]`